main.cpp

```diff
@@ -458,6 +478,19 @@ Library::Library()
     lateFees = 2.5f;
 }
 
+std::string buyNewBooks (int numNewBooks) //While #2
```

Comment:
That's a nice **free** function you have created here!
It's a shame it's using variables that aren't declared inside it.
It's also interesting that you have a member function declared in `Library`, but not implemented anywhere
🤔 🤔 🤔 🤔 
- [ ] revise

Reply:
I don't fully understand what you mean here:
1) What is a free function?
2) I am trying to change a member variable of the same struct (that may not be what is asked here, but I would like to understand the right way to do that)
3) Yes, I declared it and defined the function, but replit showed me there was an error so I never implemented it, but was trying to fix the error- which was where I was stuck with point 2) and needed your help here!

Reply:
A free function is a function that is not a member function.  it is not part of a class.
is `buyNewBooks` supposed to be a member of your `Library` class?   if so, you better name it correctly so it is!